shared/js/ui/views/tests/generate-data.mjs

```diff
@@ -294,6 +294,9 @@ export function mockToExtensionDashboardMessage(mock) {
         },
         requestData: { requests: mock.requests },
         emailProtectionUserData: undefined,
+        fireButton: {
+            enabled: withFireButton,
```

Comment:
I noticed on the preview page that we cannot see the variant without the fire button - I think we should add another iframe and ensure we can review both side-by-side

Reply:
<img width="1793" alt="Screenshot 2023-06-30 at 09 21 00" src="https://github.com/duckduckgo/privacy-dashboard/assets/1643522/95517fdd-e9d8-44a9-8aea-8ff9feb8b00c">